crates/astria-cli/src/sequencer/account.rs

```diff
@@ -126,3 +197,7 @@ struct ArgsInner {
     /// The address of the Sequencer account
     address: Address,
 }
+
+fn mnemonic_length_value_parser() -> impl TypedValueParser {
+    PossibleValuesParser::new(["12", "24"]).map(|v| v.parse::<u8>().unwrap())
```

Comment:
No panicking, especially without a message. Have a look at this: https://docs.rs/clap/latest/clap/struct.Command.html#method.error